webapp/loggedin/navigation/components/assessment/header/status.js

```diff
@@ -30,7 +30,7 @@ const Status = props => {
 
   //  Hidden in public view
   if (!userInfo) {
-    return null
+    return <div/>
```

Comment:
Is there a reason for div instead of null?

Reply:
yes , there muse be a dom element there. the parent has display flex

Reply:
done